README.md

```diff
@@ -186,6 +186,8 @@ should be installed.
 
 ## Building with Nix
 
+By default, Nix will build the project and its transitive dependencies from source, which can take up to an hour. We recommend setting up the binary cache at [kore.cachix.org] to speed up the build process significantly.
```

Comment:
```suggestion
By default, Nix will build the project and its transitive dependencies from source, which can take up to an hour. We recommend setting up the binary cache at [kore.cachix.org] to speed up the build process significantly.
```

Let's line-wrap this at 80 characters and provide a full link in `<https://app.cachix.org/cache/kore>`.

Reply:
`[the binary cache](https://app.cachix.org/cache/kore)`

Reply:
Resolved in e96f279